VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.html

```diff
@@ -309,6 +331,10 @@ <h2 class="govuk-heading-s">{{ 'judge-participant-status-list.observers' | trans
             {{ participant.name ?? participant.displayName }}
           </dt>
 
+          <dd class="govuk-body-m govuk-!-margin-bottom-3">
+            {{ 'interpreter-language.' + stringToTranslateId(participant.interpreterLanguage?.description) | translate }}
```

Comment:
the languages have the English and welsh, if we can't use the values from the api (provided by ref data) could we not include the values in the cy.json since we have the translations?